deepchem/feat/graph_data.py

```diff
@@ -2,73 +2,69 @@
 import numpy as np
 
 
-class MoleculeGraphData:
-  """MoleculeGraphData class
+class GraphData:
+  """GraphData class
 
-  This data class is almost same as `torch_geometric.data.Data 
+  This data class is almost same as `torch_geometric.data.Data
   <https://pytorch-geometric.readthedocs.io/en/latest/modules/data.html#torch_geometric.data.Data>`_.
 
   Attributes
   ----------
-  node_features : np.ndarray
+  node_features: np.ndarray
```

Comment:
As an additional suggestion, should we just go ahead and add a converter to a DGL graph? (https://docs.dgl.ai/en/0.4.x/api/python/graph.html). That would make our interop with PyG/DGL pretty seamless once we finish converting out models over to `GraphData`